app/views/courses/EnrollmentsView.coffee

```diff
@@ -24,7 +24,7 @@ module.exports = class EnrollmentsView extends RootView
   enrollmentRequestSent: false
 
   events:
-    'click #enroll-students-btn': 'onClickEnrollStudentsButton'
+    # 'click #enroll-students-btn': 'onClickEnrollStudentsButton' # we do not have this button
```

Comment:
If we're getting rid of it, let's make sure to remove all related code/files so as not to leave confusing remnants in the future.